### PR TITLE
release notes

### PR DESCRIPTION
**Pull request by qpetraroia**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+  * Creation of new clusters and nodepools on 1.18.
+  * CRUD operations on 1.18 clusters.
+  * Azure Support of non-Kubernetes related, platform issues. Platform issues include trouble with networking, storage, or compute running on Azure. Any support requests for K8s patching and troubleshooting will be requested to upgrade into a supported version.
+* Previous [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) deprecation was June 30th 2021. To better align with Kubernetes Upstream pod security policy (preview) deprecation will begin with Kubernetes version 1.21, with its removal in version 1.25. As Kubernetes Upstream approaches that milestone, the Kubernetes community will be working to document viable alternatives.
+* Containerd is now available in all regions.
```
Comment (keikhara) [inline CHANGELOG.md:14]: Please mention Windows containerd support on AKS is now available in all regions.

Comment (qpetraroia) [reply CHANGELOG.md]: Fixed. Thank you Keiko!

Comment (palma21) [reply CHANGELOG.md]: Should it say specifically windows containerd?

Comment (qpetraroia) [reply CHANGELOG.md]: Yep, I cleaned it up to match Keikos ask

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+  * Creation of new clusters and nodepools on 1.18.
+  * CRUD operations on 1.18 clusters.
+  * Azure Support of non-Kubernetes related, platform issues. Platform issues include trouble with networking, storage, or compute running on Azure. Any support requests for K8s patching and troubleshooting will be requested to upgrade into a supported version.
+* Previous [pod security policy (preview)](https://docs.microsoft.com/azure/aks/use-pod-security-policies) deprecation was June 30th 2021. To better align with Kubernetes Upstream pod security policy (preview) deprecation will begin with Kubernetes version 1.21, with its removal in version 1.25. As Kubernetes Upstream approaches that milestone, the Kubernetes community will be working to document viable alternatives.
+* Windows containerd support on AKS is now available in all regions.
```
Comment (palma21) [inline CHANGELOG.md:14]: This is a preview feature?

**Review by palma21 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (palma21) [reply CHANGELOG.md]: Also is there a doc? How can folks use it?

Comment (palma21) [reply CHANGELOG.md]: Found these
https://docs.microsoft.com/en-us/azure/aks/windows-container-cli#add-a-windows-server-node-pool-with-containerd-preview

File: CHANGELOG.md (hunk 2 above)
Comment (qpetraroia) [reply CHANGELOG.md:14]: Was in the previous release notes announcements. @miwithro wrote it and I think we should keep it up.

Comment (palma21) [reply CHANGELOG.md:14]: yes that's fine but it's an update to a preview feature, not really an announcement

**Review by keikhara — APPROVED** [review]

**Pull request merged**